src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Configuration.php

```diff
@@ -72,7 +72,15 @@ private function __construct(array $thresholds = [], $regex = '', $verboseOutput
             }
         }
         $this->regex = $regex;
-        $this->verboseOutput = $verboseOutput;
+
+        $this->verboseOutput = array_fill_keys(['unsilenced', 'direct', 'indirect', 'self', 'other'], true);
```

Comment:
Generally speaking, this still has runtime overhead, whereas declaring the resulting array is free. Also, I doubt this is more readable since it requires knowing what `array_fill_keys()` does.